### DISCUSSION TITLE
Text Color

### DISCUSSION THREAD

**Discussion by ChristianCaracach-velaONE** [rsuite/rsuite#4000] [Q&A] [answered]
Hi guys! I've been working with this library for 2 years now, I really loved when you added <Text> and <Heading>, I found them really useful, but the thing is that sometimes I need to use the "primary" color on those texts (the one from theme) and I didn't found a good way to do it. I keep falling into using a "text-primary" classname. 
Is there a way to do that?

Thanks!

**Comment by simonguo** [comment]
Thank you for your continued support of React Suite.
- Can you describe your use case in more detail? Or you can provide a simple code snippet so that we can better understand your needs.
- Do you have any suggestions for use or other related examples to refer to?

**Comment by ChristianCaracach-velaONE** [comment]
Ooops, apparently the tags I wrote didn't appear. My bad. 
So, every time I use Text tag it seems something like this:
![imagen](https://github.com/user-attachments/assets/8624bba6-7b45-4884-b3ed-95b15e98dd23)
And that returns a text with the RED color that we defined in our theme/palette (that's great).

But, there are some cases in our apps where we want to use Text with Primary color (also defined in theme/palette). And there isn't a way to do it properly, at least from RSuite API, so, I keep falling into use a handmade classname, like this:
![imagen](https://github.com/user-attachments/assets/36b064fa-cac4-4c0a-98ae-276105047a6c)

My question is: is there a way to do it better? if not, wouldn't be a good idea if we add that feature?

**Comment by ChristianCaracach-velaONE** [comment]
Hey @simonguo Is there any answer for this? Can I help with something else?

**Reply by simonguo** [reply, marked as answer]
There is currently no ideal API, you can temporarily use the CSS variable primary color.

```tsx
<Text style={{color:'var(--rs-primary-500)'}}>text</Text>
```

**Reply by ChristianCaracach-velaONE** [reply]
Ok. Thanks. I'll mark this as closed
